Server/BackEndMonitoringServer/src/CService.cpp

```diff
@@ -44,6 +42,8 @@ bool CService::Run()
 
 #elif __linux__
 
+    m_p_service = this;
```

Comment:
Move 'm_p_service = this; ' out of #ifdef scope

Reply:
Done

Reply:
Done